### ISSUE TITLE
训练模型中途报错

### ISSUE THREAD

**Issue by leechenggg** [PaddlePaddle/PaddleX#1550]
## 描述问题
用paddlex--ppyolov2 训练检测模型时中途报错，我用的自己的数据集VOC格式

## 复现
[https://aistudio.baidu.com/aistudio/projectdetail/3980820](url)

1. 您使用的命令是？
`num_classes = len(train_dataset.labels)
model = pdx.det.PPYOLOv2(num_classes=num_classes, backbone='ResNet50_vd_dcn')
model.train(
    num_epochs=160,
    train_dataset=train_dataset,
    train_batch_size=8,
    eval_dataset=eval_dataset,
    learning_rate=0.001 / 8,
    warmup_steps=1000,
    warmup_start_lr=0.0,
    save_interval_epochs=20,
    lr_decay_epochs=[216, 243, 275],
    save_dir='output/ppyolov2_r50vd_dcn')`

3. 您使用的数据集是？

数据集是自己制作的VOC格式，包含251张500*500像素的图像，共4种缺陷。

4. 请提供您出现的报错信息及相关log
![12](https://user-images.githubusercontent.com/100515681/179233245-5184abee-b983-4daa-a2d6-51198c12cbfd.jpg)

## 环境

1. 请提供您使用的PaddleX的版本号

2.0.0

3. 请提供您使用的操作系统信息，如Linux/Windows/MacOS

Windows
4. 请问您使用的Python版本是

3.7

**Comment by lailuboy** [comment]
看起来可能是数据集有问题，错误信息请再给全一点
